packages/retail-ui-extensions-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/retail-ui-extensions-react",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
why do we need to version dump? I thought we only need to do that when we publish our library.